frontend/tests/components/profile/profile-content.test.tsx

```
@@ -0,0 +1,698 @@
+// Mock DOMPurify
+jest.mock("dompurify", () => ({
+  sanitize: (html: string) => html,
+}));
+describe("ProfileContent", () => {
+  const mockUserData: AuthorizedUser = {
+    id: 1,
+    email: "test@northeastern.edu",
+    firstName: "John",
+    lastName: "Doe",
+    bio: "<p>Test bio</p>",
+    profilePhoto: null as any,
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)